applications/serial_lte_modem/src/main.c

```diff
@@ -218,15 +218,18 @@ void handle_mcuboot_swap_ret(void)
 
 	fota_stage = FOTA_STAGE_COMPLETE;
 	switch (type) {
-	/** Swap failed because image to be run is not valid */
-	case BOOT_SWAP_TYPE_FAIL:
-		LOG_INF("BOOT_SWAP_TYPE_FAIL");
+	/** Attempt to boot the contents of slot 0. */
+	case BOOT_SWAP_TYPE_NONE:
+	/** Swap to slot 1. Absent a confirm command, revert back on next boot. */
```

Comment:
```suggestion
	/** Swap to slot 1. Without a confirm command, revert back on next boot. */
```

Reply:
Please allow me to keep this as it's copied from MCUBOOT [source code](https://github.com/nrfconnect/sdk-zephyr/blob/master/include/dfu/mcuboot.h#L34)